### ISSUE TITLE
Graphics

### ISSUE THREAD

**Issue by Luc-Saccoccio** [ocaml-cross/opam-cross-windows#197]
I tried using this variant with graphics, I endend up having this error message : 
```
┌─<luc@Bouquinbourg in ~/repos/perso/AtomasOCaML>
└─<git@AtomasOCaML:master>──» ocamlfind -toolchain windows ocamlopt -linkpkg -package graphics -package unix atomas.ml -o atomas
File "atomas.ml", line 1:
Error: Files /home/luc/.opam/4.11.1+32bit/windows-sysroot/lib/graphics/graphics.cmxa
       and /home/luc/.opam/4.11.1+32bit/windows-sysroot/lib/ocaml/stdlib.cmxa
       make inconsistent assumptions over implementation Stdlib__sys
```

Is there a way to fix this ?

**Comment by alexblanche** [comment]
I had a similar issue with the Graphics package.
Ocamlfind successfully finds the Graphics module when **not** using the windows toolchain, but when I used `ocamlfind -toolchain windows ocamlopt -linkpkg -package graphics ...`, I got an "Unbound module Graphics" error.

Out of despair, I naively tried to copy the content of my folder "/home/alex/.opam/4.14.1/lib/graphics/" to "/home/alex/.opam/4.14.1/windows-sysroot/lib/graphics/", and I now have the same message.

```
ocamlfind -toolchain windows ocamlopt -linkpkg -package graphics test_graphics_aux.ml test_graphics.ml -o gra.exe
File "test_graphics.ml", line 1:
Error: Files /home/alex/.opam/4.14.1/windows-sysroot/lib/graphics/graphics.cmxa
       and /home/alex/.opam/4.14.1/windows-sysroot/lib/ocaml/stdlib.cmxa
       make inconsistent assumptions over implementation Stdlib__Sys
```

Anyway, I am stuck.
Any idea why Ocamlfind does not find the Graphics package when using the toolchain?
